tools/protobuf-gen/Cargo.toml

```diff
@@ -1,13 +0,0 @@
-[package]
-name = "protobuf-gen"
-version = "0.1.0"
-authors = ["Edouard Oger <eoger@fastmail.com>"]
-edition = "2021"
-license = "MPL-2.0"
-
-[dependencies]
-clap = "2.34"
-prost-build = "0.12"
-serde = "1"
-serde_derive = "1"
-toml = "0.5"
```

Comment:
@mhammond   Should I have deleted `protobuf-gen`?  I think we were talking a bit about trying to extend viaduct to support ohttp.  If we did that, I think we would need this back.  I'm really hoping we can take the UniFFI route instead though.

Reply:
I agree with what you said - I guess the answer to your question depends on the success of our hopes for the future here :)